lambda/models/domain_objects.py

```diff
@@ -42,11 +42,13 @@ def __str__(self) -> str:
         """Represent the enum as a string."""
         return str(self.value)
 
-    CREATING = "CREATING"
-    ACTIVE = "ACTIVE"
-    INACTIVE = "INACTIVE"
-    UPDATING = "UPDATING"
-    DELETING = "DELETING"
+    Creating = "Creating"
```

Comment:
I just meant the values, my bad. The enums themselves can/should be all caps, but the values would need to be the human readable ones. same for the other enums that you were changing.